src/cms/api/protected-area/custom/protected-area-status.js

```diff
@@ -227,11 +227,12 @@ const getProtecteAreaStatus = async (ctx) => {
       isReservationsAffected: publicAdvisory.reservationsAffected,
       eventType: publicAdvisory.eventType,
       //facilitiesCampfiresInd: getCampfireFacility(protectedArea.facilities),
-      hasCampfireBan: hasCampfireBans,
-      accessStatusEffectiveDate: protectedArea.fffectiveDate,
-      accessStatusRescindedDate: null,
+      hasSmokingBan: protectedArea.hasSmokingBan,
+      hasCampfireBan: protectedArea.hasCampfireBan,
       campfireBanEffectiveDate: null,
       campfireBanRescindedDate: null,
+      accessStatusEffectiveDate: protectedArea.fffectiveDate,
```

Comment:
is this a typo? effectiveDate?

Reply:
thanks Siva, typo and a wrong content type, I've fixed it